stock_mts_mto_rule/view/warehouse.xml

```diff
@@ -6,7 +6,7 @@
         <field name="model">stock.warehouse</field>
         <field name="inherit_id" ref="stock.view_warehouse"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='delivery_steps']" position="after">
+            <xpath expr="//field[@name='partner_id']" position="after">
```

Comment:
Why this change ?

Reply:
if you install this all by itself, the checkbox field is not displayed, that's why I say this modification as needed